GooglePlayInstant/Editor/PlayInstantRunner.cs

```diff
@@ -31,6 +31,12 @@ public static class PlayInstantRunner
         /// </summary>
         public static void BuildAndRun()
         {
+            if(AndroidSdkManager.AndroidSdkRoot == null)
+            {
+                PlayInstantBuilder.LogError("Build and Run failed to locate the Android SDK. Check Preferences -> External Tools for the path.");
```

Comment:
The title will already say Build Error. So more concise "Failed to locate the Android SDK. Check Preferences -> External Tools to set the path."